Project.toml

```diff
@@ -83,7 +84,7 @@ LinearAlgebra = "1"
 LinearMaps = "2.7, 3.0"
 LoopVectorization = "0.12.151"
 MPI = "0.20.6"
-Makie = "0.19, 0.20"
+Makie = "0.21"
```

Comment:
Was this required due to the update to Julia 1.10?

Reply:
Yes

Reply:
https://github.com/trixi-framework/Trixi.jl/actions/runs/12220953105/job/34089422142

Reply:
> Was this required due to the update to Julia 1.10?

I can only confirm that it is required due to the update of SciMLBase.jl to 2.67.0

Reply:
https://github.com/MakieOrg/Makie.jl/blob/v0.20.10/Project.toml v0.20.10 is compatible with Julia 1.3

Reply:
Thanks!